VideoWeb/VideoWeb/ClientApp/src/app/shared/models/participant-contact-details.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { ConferenceTestData } from 'src/app/testing/mocks/data/conference-test-data';
+import { ParticipantContactDetails } from './participant-contact-details';
+
+describe('ParticipantContactDetails', () => {
+    it('should map participant info', () => {
+        const participants = new ConferenceTestData().getListOParticipantContactDetailsResponseVho(
+            'C7163972-A362-4167-8D33-77A64674B31C',
+            'MyVenue'
+        );
+        const participant = participants[0];
+        const p = new ParticipantContactDetails(participant);
+        p.judgeInAnotherHearing = true;
```

Comment:
This is literally cheating. You've set line 12 to true and then expect it to be true. Better to set this value in the `ParticipantContactDetailsResponseVho` object

Reply:
Its not cheating

Reply:
good catch though

Reply:
done